ecosystem/sep-0039.md

```diff
@@ -0,0 +1,274 @@
+## Preamble
+
+```
+SEP: 0039
+Title: Signed Payload Strkeys
+Author: David Mazières, Tomer Weller <@tomerweller>, Leigh McCulloch <@leighmcculloch>, Alfonso Acosta <@2opremio>
+Track: Standard
+Status: Active
+Created: 2019-09-16
+Updated: 2021-07-20
+Version: 0.0.1
+Discussion: https://groups.google.com/g/stellar-dev/c/Wp7gNaJvt40
+```
+
+## Simple Summary
+
+Strkey is an ASCII format for representing Stellar account IDs and
+addresses. This document introduces a new strkey version to represent
+ed25519 signed payload signers introduced by [CAP-40].
+
+## Motivation
+
+[CAP-40] introduced a new signer for disclosing transaction signatures
+in transactions. Some ecosystem services, such as Horizon, only refer to
+signers in string form. Stellar SDKs also use strings to pass around
+signers. A string form for the signer introduced by CAP-40 is required
+for these services.
+
+## Abstract
+
+A new strkey format is introduced to represent both the public key and
+variable length payload of a ed25519 signed payload signer, as specified
+in [CAP-40].
+
+## Specification
+
+Strkey, the ASCII encoding for accounts, keys, and signers, currently
+covers covered ED25519 public keys, ED25519 private keys (also known as
+seeds), pre-authorized transaction hashes, hash-x signers (which provide
+signing authority upon revelation of a SHA-256 preimage), and muxed
+accounts.
+
+One change is made to add the new signer key type,
+STRKEY_SIGNED_PAYLOAD, which encodes a public key followed by a variable
+length payload that is prefixed with a 4 byte length.
+
+| Key type                | Base value   | First char | Muxed | Alg  |
+| ----------------------- | ------------ | ---------- | ----- | ---- |
+| STRKEY\_PUBKEY          | 6 << 3       | G          | no    | PK   |
+| STRKEY\_MUXED           | 12 << 3      | M          | yes   | PK   |
+| STRKEY\_PRIVKEY         | 18 << 3      | S          | no    | PK   |
+| STRKEY\_PRE\_AUTH\_TX   | 19 << 3      | T          | no    | Hash |
+| STRKEY\_HASH\_X         | 23 << 3      | X          | no    | Hash |
+| STRKEY\_SIGNED_PAYLOAD  | 23 << 3      | P          | no    | PK   |
```

Comment:
I think you mean 15 << 3 (see https://datatracker.ietf.org/doc/html/rfc4648#section-6)

Reply:
Yes I do. Thanks.